src/tribol/geom/GeomUtilities.cpp

```diff
@@ -1137,10 +1126,9 @@ bool SegmentIntersection2D( const real xA1, const real yA1, const real xB1, cons
      yDiff = (yDiff < 0.) ? -1.0 * yDiff : yDiff;
 
      real diffTol = 1.0E-3;
-     SLIC_ERROR_IF( xDiff > diffTol || yDiff > diffTol, 
+     SLIC_DEBUG_IF( xDiff > diffTol || yDiff > diffTol, 
                    "SegmentIntersection2D(): Intersection coordinates are not equally derived." );
```

Comment:
Future -- this message would likely be more useful w/ more context and/or actual values

Reply:
True. I'll hold off for now. This is really more of a math sanity check and may not necessarily need to be checked. It has never presented an issue.